docs/_docs/04_community/contributing.md

```diff
@@ -150,16 +149,50 @@ to include the following:
    test output so we can verify that everything is working.
 1. Any notes on backwards incompatibility or downtime.
 
+#### Validate the Pull Request for Azure Platform
+
+After [creating a pull request](https://help.github.com/articles/creating-a-pull-request/), please follow the below guideline for [Azure Platform](https://azure.com)
+
+##### To run the CI Pipeline on forked repo
+
+1. Run the following command on your preffered Terminal and copy the output
+
+```bash
+az ad sp create-for-rbac --name "terratest-az-cli" --role contributor --sdk-auth
+```
+
+2. Go to Secrets page ([https://github.com/<GITHUB_ACCOUNT>/terratest/settings/secrets](https://github.com/<GITHUB_ACCOUNT>/terratest/settings/secrets)) under Settings tab ([https://github.com/<GITHUB_ACCOUNT>/terratest/settings](https://github.com/<GITHUB_ACCOUNT>/terratest/settings)) on GitHub

```

Comment:
NIT: to make the numbered list easier to maintain, could you change these all to `1.`? Markdown will automatically replace them with the proper numbers.

```suggestion
1. Go to Secrets page ([https://github.com/<GITHUB_ACCOUNT>/terratest/settings/secrets](https://github.com/<GITHUB_ACCOUNT>/terratest/settings/secrets)) under Settings tab ([https://github.com/<GITHUB_ACCOUNT>/terratest/settings](https://github.com/<GITHUB_ACCOUNT>/terratest/settings)) on GitHub

```

Reply:
thanks for the suggestion, I just changed list numbers with `1`.

also ran the `MarkdownLinter` and removed couple of empty lines from the file.